dev-util/bcc/bcc-0.22.0-r1.ebuild

```
@@ -19,15 +18,17 @@ KEYWORDS="~amd64 ~arm64 ~x86"
 IUSE="+lua test"
 REQUIRED_USE="${PYTHON_REQUIRED_USE}
 	lua? ( ${LUA_REQUIRED_USE} )"
+# tests need root access
+RESTRICT="test"
 
 RDEPEND="
-	~dev-libs/libbpf-0.3:=[static-libs(-)]
-	>=sys-kernel/linux-headers-4.14
 	>=dev-libs/elfutils-0.166:=
-	<=sys-devel/clang-13:=
-	<=sys-devel/llvm-13:=[llvm_targets_BPF(+)]
-	lua? ( ${LUA_DEPS} )
+	>=dev-libs/libbpf-0.5.0:=[static-libs(-)]
+	>=sys-kernel/linux-headers-5.13
+	<=sys-devel/clang-14:=
```

I prefer:
```suggestion
	<sys-devel/clang-$((${LLVM_MAX_SLOT} + 1)):=
```
.. and so on. Then just define LLVM_MAX_SLOT at the very top of the ebuild above inherit for cleanliness.

---

Oh yes, this looks cleaner, thanks